client/client.go

```diff
@@ -535,6 +539,31 @@ func (c *Client) Collect(ch chan<- prometheus.Metric) {
 	}
 }
 
+// argListFromPgParams creates an array of strings from the pointer to the parameters for pt-pg-sumamry
+func argListFromPgParams(pParams *agentpb.StartActionRequest_PTPgSummaryParams) []string {
+	var args []string
+
+	// Only adds the arguments are valid
+
+	if pParams.Host != "" {
+		args = append(args, "--host", pParams.Host)
+	}
+
+	if pParams.Port > 0 && pParams.Port <= 65535 {
+		args = append(args, "--port", strconv.FormatUint(uint64(pParams.Port), 10))
```

Comment:
Wasn't fmt.Sprintf shorter?

Reply:
In my opinion sprintf has more overhead because it has to parse the string first. This is just cast to 64 bits and then direct conversion to string.
Do you want me to change it?

Reply:
let's use `strconv.Itoa` fot better readability.